site/docs/v1/tech/identity-providers/_idp-managed-domains-troubleshooting.adoc

```diff
@@ -0,0 +1,25 @@
+A common pattern is to set up {idp_type_name} as the login method for the FusionAuth application. This lets you keep all your users in the {idp_type_name} system, and have FusionAuth delegate auth decisions to it. However, if your Identity Provider isn't configured correctly, you can end up in a dead end, unable to log in. This might happen if you:
+
+* Set up {idp_type_name} as the authentication method for your FusionAuth admin application.
+* Set the [field]#Managed domains# to your domain. For example, `example.com`.
+* Don't have any users with a domain that is different than `example.com`.
+* Accidentally misconfigure {idp_type_name} and break authentication.
+* Log out of the FusionAuth admin application.
+
+You won't be able to log in to the FusionAuth admin application to correct the misconfiguration. You've locked down the FusionAuth admin application so that only valid logins can access it, but because of the misconfiguration, there is no account with a valid {idp_type_name} login.
+
+image::troubleshooting/unable-to-login-managed-domains.png[Unable to login with a managed domain and misconfigured Identity Provider.,width=1200,role=bottom-cropped]
+
+This is a problem, but not an insurmountable one. Your options depend on when you discover the issue. If you are beginning your {idp_type_name} configuration, you can avoid this scenario. Follow these steps:
```

Comment:
We could also suggest they ensure they have an API key created during configuration so they have some additional options later.